### PR TITLE
Replace selectisize tom select

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Review by escattone — COMMENTED** [review]
❤️ it! Nice find! It looks great except for one thing. For me,  the `x` of the clear button isn't visible (see below):

image

It looks to me like the SCSS for the clear and remove button plugins is being added to the `screen.scss`, so I'm confused why it isn't visible.

File: kitsune/sumo/static/sumo/js/wiki.editor.js [hunk 1]
```diff
@@ -173,34 +162,49 @@ function clearSelected(name) {
...
-  ).addEventListener("click", function(event) {
+  ).addEventListener("click", function (event) {
     event.preventDefault();
     clearSelected("products");
   });
+
+  new TomSelect("select[id='id_restrict_to_groups']", {
+    closeAfterSelect: true,
+    plugins: {
+      clear_button: {
```
Comment (akatsoulas) [inline kitsune/sumo/static/sumo/js/wiki.editor.js:199]: @escattone `tom-select` doesn't have an `x` for the clear button but rather a checkbox. 
See https://tom-select.js.org/plugins/clear-button/

We could override this but unless there are strong objections I would prefer to tackle this in a different PR/issue to avoid scope creep. Thoughts?

Comment (escattone) [reply kitsune/sumo/static/sumo/js/wiki.editor.js:199]: Yes, let's tackle that in another PR/issue!

However, I do see a small `x` both in our HTML and on their website.
image

**Review by escattone — APPROVED** [review]
r+wc

File: kitsune/sumo/static/sumo/js/wiki.editor.js (hunk 1 above)
Comment (akatsoulas) [reply kitsune/sumo/static/sumo/js/wiki.editor.js:199]: that's for Bootstrap 4/5. If you select the `Default` from the drop down it's the checkbox..

**Comment by emilghittasv** [comment]
> ❤️ it! Nice find! It looks great except for one thing. For me, the `x` of the clear button isn't visible (see below):
> image
> 
> It looks to me like the SCSS for the clear and remove button plugins is being added to the `screen.scss`, so I'm confused why it isn't visible.

I've also encountered this issue during testing in `stage`. Filled under https://github.com/mozilla/sumo/issues/2029
